### ISSUE TITLE
The colab notebook is broken again

### ISSUE THREAD

**Issue by Blazeolmo** [minimaxir/aitextgen#222]
Errors when running the first cell. i am impressed, really, that this time it broke so hard not even the first cell works.
I wonder what this project could be if it wasn't made by just a random dude...

**Comment by atari486** [comment]
Yeah, no one (least of all the developer) has any vested interest in helping. The courtesy of backwards compatibility is long gone, so you can't expect it. Nobody cares anymore. All people care about is being able to do the next new shiny thing, even if it breaks everything that already exists.

With that being a reality, I recommend trying to get in with the free machines on Paperspace instead. They haven't updated all their Pythons/packages (yet) so it'll work if you make some slight modifications (pay attention to the error messages). Of course, advertising this will probably make it even harder to snag one of the free instances now, but better some chance than none.

**Comment by tommysteryy** [comment]
If you're looking for, essentially the same notebook, but one that works, I recommend the one linked here instead: https://minimaxir.com/2019/09/howto-gpt2/. This one works totally fine for me on Google Collab and it practically is the same notebook just not using `aitextgen`. Of course, you lose the other functionality in `aitextgen`, but it's a good start.

**Comment by atari486** [comment]
Thank you Tommy. This is just what I needed, and I can confirm that it works.

Yet I still see this:
- In the glory days of textgenrnn, we could set anything - dropout rate, RNN size/layers, etc.
- With the switch to aitextgen, we lost many of those options.
- Now, you can't even continue training without restarting the runtime.

This is not progress. As time goes on, the users should have more control, not less.
